personalInfoDashboard.js

```diff
@@ -0,0 +1,73 @@
+// information
+const firstName = "Mariam";
+const lastName = "Poghosyan";
+const birthYear = 2003;
+const currentYear = 2025;
+const isStudent = false;
+const hobbies = ['listening to music', 'hiking', 'Armenian folk dances']
+let contact = {
+  email: 'mariam.poghosyan.techx@gmail.com',
+  phone: '+374 93 22-07-67',
+  city: 'Yerevan',
+};
+
+console.log(`Hi, my name is ${firstName} ${lastName}. I live in ${contact.city}.`);
+console.log(`I am ${currentYear - birthYear} years old and currently ${isStudent ? "a student" : "not a student"}.`);
+
+//  Type Coercion & Comparison
+
+ageString = "25";
+ageNumber = 25;
+
+ageString == ageNumber;
+ageString === ageNumber;
+
+console.log("comparison with ==", ageString == ageNumber);
+console.log("comparison with ===", ageString === ageNumber);
+
+console.log("Type of string:",typeof ageString);
```

Comment:
fix the missing space
```suggestion
console.log("Type of string:", typeof ageString);
```

Reply:
thank you, I've fixed it.